app/controllers/users.js

```diff
@@ -19,3 +20,20 @@ exports.signUp = async (req, res, next) => {
     return next(error);
   }
 };
+
+exports.signIn = (req, res, next) => {
+  try {
+    const { user } = res.locals;
+
+    const payload = {
+      id: user.id,
+      email: user.email
+    };
+    const secret = 'hola1234';
```

Comment:
The value of SECRET must be loaded from an env var. Using the standard HSA 256 encryption for the signature, the secret should at least be 32 characters long, but the longer the better.